src/components/utils/http.js

```diff
@@ -0,0 +1,30 @@
+import { events } from 'onfido-sdk-core'
+const SDK_SERVER_URL = 'https://sdk.onfido.com'
+
+const reduceObj = (object, callback, initialValue) =>
+  Object.keys(object).reduce(
+    (accumulator, key) => callback(accumulator, object[key], key, object),
+    initialValue)
+
+const objectToFormData = (object) =>
+  reduceObj(object, (formData, value, key) => {
+    formData.append(key, value)
+    return formData;
+  }, new FormData())
+
+export const postToServer = (payload, serverUrl, token, onComplete) => {
+  const request = new XMLHttpRequest()
+  const url = serverUrl ? serverUrl : SDK_SERVER_URL
+  request.open('POST', `${url}/validate_document`)
+  request.responseType = 'json'
+  request.setRequestHeader('Content-Type', 'application/json')
+  request.setRequestHeader('Authorization', token)
+
+  request.onload = () => {
+    if (request.readyState === request.DONE) {
```

Comment:
I don't think you need this line: http://stackoverflow.com/questions/9181090/is-onload-equal-to-readystate-4-in-xmlhttprequest